.github/workflows/ci.yml

```diff
@@ -24,6 +24,10 @@ jobs:
         with:
           toolchain: ${{ matrix.version }}
           components: rustfmt, clippy
+      # tokio 1.39+ requires rust <= 1.70.0
+      # https://github.com/tokio-rs/tokio?tab=readme-ov-file#supported-rust-versions
+      - run: cargo update -p tokio --precise 1.38.1
+        if: matrix.version == '1.68.0'
```

Comment:
We might want to consider updating the `dev-dependencies` to `tokio = "=1.38.1"` rather than updating the workflow. I feel that we don’t need to use the latest tokio in the stable version either, since it’s only used for testing.

Reply:
Thank you. I understand that's one of the suggestions you gave me earlier. I thought this crate is most likely to be used with tokio (for async loading) in real case anyway, and it might be nicer to test against arbitrary (latest) versions of tokio as well - what would you think?

Reply:
Let’s focus on what we want to ensure: that our async functions work as expected on any async runtime. We don’t need to prioritize compatibility with the latest version of `tokio`.

Reply:
Ok, that's a fair point. Thank you for your feedback and handling 🙏